internal/services/portal/get_session_test.go

```
@@ -0,0 +1,60 @@
+	require.Equal(t, "user_456", info.ExternalID)
+	require.Equal(t, "pc_789", info.PortalConfigID)
+	require.Equal(t, []string{"keys:read", "analytics:read"}, info.Permissions)
+	require.Equal(t, map[string]any{"name": "Test User", "email": "test@example.com"}, info.Metadata)
+	require.True(t, info.Preview)
+}
+
+func TestSessionInfo_NilPermissionsAndMetadata(t *testing.T) {
+	t.Parallel()
+
+	info := SessionInfo{
+		WorkspaceID:    "ws_123",
```

golangci-lint (2.11.4)

[error] unusedwrite: unused write to field WorkspaceID

(govet)